resources/rollups.yaml

```diff
@@ -0,0 +1,248 @@
+        "measurement": "network/rx_rate",
+        "name": "network/rx_rate/medium",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value",
+            "alias": "value_max"
+          },
+          {
+            "function": "mean",
+            "field": "value",
+            "alias": "value_mean"
+          }
+        ]
+      },
+      {
+        "retention": "long",
+        "measurement": "cpu/usage_rate",
+        "name": "cpu/usage_rate/long",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value",
+            "alias": "value_max"
+          },
+          {
+            "function": "mean",
+            "field": "value",
+            "alias": "value_mean"
+          }
+        ]
+      },
+      {
+        "retention": "long",
+        "measurement": "memory/usage",
+        "name": "memory/usage/long",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value",
+            "alias": "value_max"
+          },
+          {
+            "function": "mean",
+            "field": "value",
+            "alias": "value_mean"
+          }
+        ]
+      },
+      {
+        "retention": "long",
+        "measurement": "filesystem/usage",
+        "name": "filesystem/usage/long",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value"
+          }
+        ]
+      },
+      {
+        "retention": "long",
+        "measurement": "network/tx",
+        "name": "network/tx/long",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value"
+          }
+        ]
+      },
+      {
+        "retention": "long",
+        "measurement": "network/tx_rate",
+        "name": "network/tx_rate/long",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value",
+            "alias": "value_max"
+          },
+          {
+            "function": "mean",
+            "field": "value",
+            "alias": "value_mean"
+          }
+        ]
+      },
+      {
+        "retention": "long",
+        "measurement": "network/rx",
+        "name": "network/rx/long",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value"
+          }
+        ]
+      },
+      {
+        "retention": "long",
+        "measurement": "network/rx_rate",
+        "name": "network/rx_rate/long",
+        "functions": [
+          {
+            "function": "max",
+            "field": "value",
+            "alias": "value_max"
+          },
+          {
+            "function": "mean",
+            "field": "value",
+            "alias": "value_mean"
+          }
+        ]
+      },
+      {
+        "retention": "medium",
+        "measurement": "system",
+        "name": "load1/medium",
+        "functions": [
+          {
+            "function": "mean",
+            "field": "value",
```

Comment:
Won't trailing commas here and below mess anything up?

Reply:
I exported/imported that json before updating this file. It should work!

Reply:
Oh, you are right. It is rollups file. Let me fix that.